crates/iceberg-catalog/migrations/20240620151544_views.sql

```diff
@@ -38,14 +38,15 @@ create table view_schema
 call add_time_columns('view_schema');
 select trigger_updated_at('view_schema');
 
+
+
 create table view_version
 (
-    view_version_uuid    uuid        not null primary key default uuid_generate_v1mc(),
-    view_id              uuid        not null REFERENCES view (view_id) ON DELETE CASCADE,
-    version_id           bigint      not null,
-    schema_id            int         not null,
-    timestamp            timestamptz not null,
-    default_namespace_id uuid REFERENCES namespace (namespace_id),
+    view_version_uuid uuid   not null primary key default uuid_generate_v1mc(),
```

Comment:
Most keys exist twice, once as non-globally unique int as they are used in java and then again we need a unique pkey which are uuids, to avoid confusion I suffixed them with _uuid, I'm happy with other solutions. It's a bit unfortunate that the version id, schema id etc are not globally unique.